### PR TITLE
feat: Adds checkstyle module.

### PR DESCRIPTION
**Pull request by damencho**

### CODE REVIEW

File: src/main/java/org/ice4j/stack/StunServerTransaction.java [hunk 1]
```diff
@@ -165,7 +165,8 @@ public void sendResponse(Response         response,
                IOException,
                IllegalArgumentException
     {
-        if(!isRetransmitting){
+        if(!isRetransmitting)
```
Comment (JonathanLennox) [inline src/main/java/org/ice4j/stack/StunServerTransaction.java:168]: Don't we normally have a space after `if`?  Is checkstyle not catching this?

**Comment by ibauersachs** [comment]
I suggested this before to @damencho not sure if it made it to everyone: https://github.com/google/google-java-format

I know it would be a radical change, but it would squash any discussions about style. Apply the very strict formatter, enforce it with a Maven goal and you're done. The IntelliJ plugin does everything for you with Ctrl+Alt+L.

**Comment by damencho** [comment]
> I suggested this before to @damencho not sure if it made it to everyone: https://github.com/google/google-java-format
> 
> I know it would be a radical change, but it would squash any discussions about style. Apply the very strict formatter, enforce it with a Maven goal and you're done. The IntelliJ plugin does everything for you with Ctrl+Alt+L.

Yeah, that is indeed very radical change, it is even changing the line length to 100.

**Comment by ibauersachs** [comment]
> Yeah, that is indeed very radical change, it is even changing the line length to 100.

So what?

(The Kotlin code uses longer lines as well).

**Comment by damencho** [comment]
Its a step backwards from 120.

**Review by bgrozev — APPROVED** [review]

**Pull request merged**